_config.yml

```diff
@@ -73,7 +73,10 @@ title: "Workshop Title"
 # Generic settings (should not need to change).
 #------------------------------------------------------------
 
-# What kind of thing is this ("workshop" or "lesson")?
+# The setting kind takes configurations from the style template.
+# It has options of "workshop" or "lesson".  Even if you are
+# teaching a lesson, choose "workshop" and use a custom
+# schedule as the option "lesson" will not build.
```

Comment:
That seems ok.  It would be good to have some information about he meaning of "lesson". Perhaps "custom set of lessons," should be "custom set of lesson or a single lesson,"